subsys/fb/cfb.c

```diff
@@ -218,6 +218,10 @@ int cfb_framebuffer_clear(const struct device *dev, bool clear_display)
 	desc.pitch = fb->x_res;
 	memset(fb->buf, 0, fb->size);
 
+	if (clear_display) {
```

Comment:
From the viewpoint of API behavior consistency, it is not preferable to finalize.
(All the other APIs that excluding finalize not update the screen.) 
IMO, It is better to remove the argument.

Reply:
Isn't it less critical to implement the behavior as documented in CFB's API rather than introducing a non-backward compatible change to CFB's API.